packages/table/src/table.tsx

```diff
@@ -520,6 +520,7 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
 
     public constructor(props: ITableProps, context?: any) {
         super(props, context);
+        this.validateProps(props, true);
```

Comment:
why is this necessary? `AbstractComponent` should be doing this.

Reply:
see https://github.com/palantir/blueprint/pull/3702#discussion_r326826457